app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  def update_likes_counter
+    post.increment!(:likesCounter)
+  end
```

Comment:
## Optional
- [ ] I suggest that you use one of the Rails Active Callbacks - `after_save` to call the `#update_likes_counter`  every time a post receive a like, then make the `update_likes_counter` method private